t/fixtures/proxy.lua

```diff
@@ -0,0 +1,222 @@
+require('resty.core')
+
+local http = require('resty.http')
+local ngx_re_match = ngx.re.match
+local str_lower = string.lower
+local tonumber = tonumber
+local format = string.format
+local ceil = math.ceil
+
+local inclusive = { inclusive = true }
+
+local cr_lf = "\r\n"
+
+local http_ok = 'HTTP/1.1 200 OK' .. cr_lf
+
+
+ngx.log(ngx.INFO, 'proxy starting')
+
+local function send(socket, data)
+--    if ngx.ctx.upstream == socket then
```

Comment:
Should we delete this?

Reply:
It is testing/debugging code. I rather keep it because it is easy to uncomment when debugging.